packages/dd-trace/src/plugins/tracing.js

```diff
@@ -46,7 +58,19 @@ class TracingPlugin extends Plugin {
     this.activeSpan.finish()
   }
 
+  asyncEnd (...args) {
+    this.finish(...args)
+    this.exit(...args)
```

Comment:
This should only happen for `OutgoingPlugin` and `InternalPlugin`. Even though it doesn't really affect this PR since it only updates an internal plugin, it should still be moved to the right place to avoid issues with other plugins as they are updated to use the new event.

Reply:
@rochdev There's nothing currently called `InternalPlugin`, AFAIK.